json-schema-generator/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id("io.micronaut.build.internal.json-schema-module")
+}
+
+dependencies {
+    compileOnly(mn.micronaut.core.processor)
+
+    implementation(mnSourcegen.micronaut.sourcegen.model)
+    implementation(mnSourcegen.micronaut.sourcegen.generator)
+    implementation(mnSourcegen.micronaut.sourcegen.generator.java)
+    implementation(mnSourcegen.micronaut.sourcegen.annotations)
+    implementation(mnSerde.micronaut.serde.jackson)
+
+    api(projects.micronautJsonSchemaAnnotations)
+    api(projects.micronautJsonSchemaCommon)
+    api(mn.jackson.databind)
```

Comment:
why are we depending on Jackson databind?

Reply:
I guess it was an old dependency, removing it now. 👍